src/backend/lib/upgrade-download.js

```diff
@@ -276,6 +282,8 @@ class Check extends EventEmitter {
 
     checkLog("checking for an upgrade..");
     this.storage.getAvailableUpgrades((err, options) => {
+      // Seriously fatal error, if upgrade storage ever becomes corrupted
+      // in the future this might warrant a way for the frontend to purge storage to reset it?
```

Comment:
Ok, so if the updater is in this state, then we consider the updater to be broken, until we have a way for the user to resolve this.